tests/test_app.py

```diff
@@ -56,9 +57,12 @@ def test_whole_audit_flow(client):
     assert result2["riskLimit"] == 5
 
 def setup_whole_audit(client, election_id, name, risk_limit, random_seed):
-    contest_id = str(uuid.uuid4())
+    contest_id_1 = str(uuid.uuid4())
```

Comment:
I did the bare minimum to run a test with the multiple contests. 
Stopped fixing tests (which now break) and making code changes when I got to manifests & results.